lib/ticks-to-tree.js

```diff
@@ -10,19 +14,24 @@ const nodeBootstrapRegExp = / internal\/bootstrap.+\.js:\d+:\d+$/
 
 module.exports = ticksToTree
 
-function ticksToTree (ticks, options = {}) {
-  const { mapFrames, inlined, pathToNodeBinary = process.execPath } = options
-  const merged = {
-    name: 'all stacks',
-    value: 0,
-    top: 0,
-    children: []
-  }
-  const unmerged = {
-    name: 'all stacks',
-    value: 0,
-    top: 0,
-    children: []
+let globalOptions
```

Comment:
can you please explain the rationale behind this?

if we need to maintain singleton state, do it at the top level 

options can be passed down through functions – can we do that instead?

Reply:
Yes, looking at it again I am not sure why I even used that globalOptions variable as it only needs to pass to the renameFrame method